### PR TITLE
2.4.1

### PR DESCRIPTION
**Pull request by fy1900**
Added 6 new icons

### CODE REVIEW

File: docs/keywords.json [hunk 1]
```diff
@@ -4561,5 +4561,17 @@
     ],
     "category":"Charts and Widgets",
     "release":"2.3.5"
-  }
+  },
+  "no-map":{
+    "alias": [
+      "map"",
```
Comment (paulcpederson) [inline docs/keywords.json]: extra `"` on this line I think is screwing up the rest of this entry

Comment (fy1900) [reply docs/keywords.json]: Thanks a lot. I've updated the code. Please review again.

**Comment by paulcpederson** [comment]
@fy1900 Great! Can you do `npm run optimize` so that the ids are removed from the svgs / they get added to the JavaScript lib?

Also, if you're bumping the version can you also add a note about these icons to the changelog?

**Comment by fy1900** [comment]
@paulcpederson The icons have been optimized. Can you please review them again?

**Comment by KatelynSeitz** [comment]
This with Griff's sprites look good.

**Comment by paulcpederson** [comment]
:+1: thanks @fy1900

**Comment by fy1900** [comment]
Thanks a lot for your help.

**Pull request merged**